tests/api_tests/nodes/test_views.py

```diff
@@ -1086,6 +1086,24 @@ def test_partial_update_private_project_logged_in_non_contributor(self):
         assert_equal(res.status_code, 403)
         assert_in('detail', res.json['errors'][0])
 
+    def test_multiple_patch_requests_with_same_category_generates_one_log(self):
+        self.private_project.category = 'project'
+        self.private_project.save()
+        new_category = 'data'
+        payload = make_node_payload(self.private_project, attributes={'category': new_category})
+        original_n_logs = len(self.private_project.logs)
+
+        res = self.app.patch_json_api(self.private_url, payload, auth=self.user.auth)
```

Comment:
For clarity, please add a check that the return code after each request is also what you'd expect.

(the lack of `expect_errors` implies the request succeeded, but it's good practice to explicitly ensure that tests work for the reason we expect)